Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionDrawerBase.cs

```diff
@@ -22,12 +26,20 @@ public override bool CanCacheInspectorGUI(SerializedProperty property)
 
         public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
         {
+            // don't use the custom property drawer when multi-editing for now. Just show the default
```

Comment:
Ideally should contain why we do that.

Reply:
Actually multi-editing seems to be working now, so I've taken this out.